sigstore/_cli.py

```diff
@@ -275,6 +275,17 @@ def _parser() -> argparse.ArgumentParser:
     )
 
     verification_options = verify.add_argument_group("Extended verification options")
+    verification_options.add_argument(
+        "--certificate-chain",
+        metavar="FILE",
+        type=argparse.FileType("rb"),
+        help=(
+            "Path to a list of CA certificates in PEM format which will be needed when building "
+            "the certificate chain for the signing certificate; must start with the parent "
+            "intermediate CA certificate of the signing certificate and end with the root "
+            "certificate"
```

Comment:
I might be wrong, but I don't think we actually require this ordering: this cert chain ends up in an `X509Store`, which AFAICT allows certs to be added in any order.

Reply:
True, I'll remove that. I just pasted that from `cosign`.

Reply:
[b3f3847](https://github.com/sigstore/sigstore-python/pull/323/commits/b3f3847d70cbd71d0523091a5460ef45a2dd11fc)